packages/sidecar/src/main.ts

```diff
@@ -1,7 +1,7 @@
-import { createWriteStream, readFile } from 'fs';
-import { IncomingMessage, Server, ServerResponse, createServer, get } from 'http';
-import { extname, join } from 'path';
-import { createGunzip, createInflate } from 'zlib';
+import { createWriteStream, readFile } from 'node:fs';
+import { IncomingMessage, Server, ServerResponse, createServer, get } from 'node:http';
+import { extname, join } from 'node:path';
+import { createGunzip, createInflate } from 'node:zlib';
```

Comment:
No objections to this but I think this is a good reminder that we haven't set the `engines` property in the `package.json`. So technically, we don't set a minimum node version. IIRC the `node:` prefixes were introduced with `14.18.0` so I'm fine with changing this now, given the age of Node 14. Setting `engines` doesn't have to happen in this PR but I think we should set it.

Reply:
source: https://nodejs.org/api/esm.html#node-imports

Reply:
Sounds good, thanks a lot! I think we are already Node v18+ or something?

Reply:
#440

Reply:
The Sentry Node SDK supports Node >= 14.18.0 but we can be more restrictive with Spotlight. Given Node 16 is EOL, 18 sounds fine to me.

Reply:
https://github.com/getsentry/sentry-javascript/blob/eb23dc4a4514b7e495af92fee9ccdfbce1dbdecc/packages/node/package.json#L9-L11